app/src/main/java/com/sunasterisk/anticovid_19/utils/JsonExtension.kt

```diff
@@ -5,16 +5,24 @@ import com.sunasterisk.anticovid_19.data.model.Country
 import com.sunasterisk.anticovid_19.data.model.Global
 import org.json.JSONArray
 import org.json.JSONException
+import org.json.JSONObject
 
 @Throws(JSONException::class)
-inline fun <reified T> String.parseJsonArray() = with(JSONArray(this)) {
-    List(length()) {
-        let { jsonArray ->
-            when (T::class) {
-                Country::class -> Country(jsonArray.getJSONObject(it)) as T
-                Global::class -> Global(jsonArray.getJSONObject(it)) as T
-                else -> throw JSONException(R.string.error_parse_json.toString())
-            }
+inline fun <reified T> String.parseJsonArray() = JSONArray(this).let { jsonArray ->
+    List(jsonArray.length()) { index ->
+        when(T::class) {
+            Country::class -> Country(jsonArray.getJSONObject(index)) as T
+            Global::class -> Global(jsonArray.getJSONObject(index)) as T
+            else -> throw JSONException(R.string.error_parse_json.toString())
         }
     }
 }
+
+@Throws(JSONException::class)
+inline fun <reified T> String.parseJsonObject() = JSONObject(this).let { jsonObject ->
+    when (T::class) {
+        Country::class -> Country(jsonObject) as T
+        Global::class -> Global(jsonObject) as T
+        else -> throw JSONException(R.string.error_parse_json.toString())
+    }
```

Comment:
```suggestion
    when (T::class) {
        Country::class -> Country(jsonObject)
        Global::class -> Global(jsonObject)
        else -> throw JSONException(R.string.error_parse_json.toString())
    } as T
```
Để như cũ vẫn được nhé